.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Join the axe Slack Community
+    url: https://accessibility.deque.com/axe-community
+    about: Connect with other people passionate about digital accessibility and axe tools.
```

Comment:
Nitpick, should these end with a newline? We do that everywhere else in the repo.